streamwebs_frontend/streamwebs/models.py

```diff
@@ -1064,3 +1064,42 @@ def __str__(self):
     class Meta:
         verbose_name = 'soil survey'
         verbose_name_plural = 'soil surveys'
+
+
+class ResourceManager(models.Model):
+    """ Manager for the Resources model """
+    def create_resource(
+        self, name, res_type, downloadable, thumbnail, sort_order
+    ):
+        return self.create(
+            name=name, res_type=res_type, downloadable=downloadable,
+            thumbnail=thumbnail, sort_order=sort_order
+        )
+
+
+class Resource(models.Model):
+    """ This model organizes Resources like pdfs or videos """
+    TYPE_CHOICES = (
+        (None, '-----'),
+        ('data_sheet', _('Data Sheet')),
+        ('publication', _('Publication')),
+        ('tutorial_video', _('Tutorial Video')),
+    )
+    name = models.CharField(max_length=255, blank=False)
+    res_type = models.CharField(
+        max_length=255, blank=False, choices=TYPE_CHOICES
+    )
+    # bad way to handle upload_to!  It should be based on res_type
```

Comment:
I think Levi meant to say that the `upload_to={ARGS}` are hard coded and should be dynamic based on the asset type. I think it's fine for now that all "downloadable" objects are in the assets files.